notebooks/en/structured_generation_vision_languag_models.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<p>Would be nice to include a link to Outlines as well here</p><p> </p><p>"We will be using the SmolVLM-Instruct model from HuggingFaceTB to extract structured information from documents, and the Outlines library for facilitating structured generation based on limiting token sampling probabilities."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>